test/ui-testing/CustomInsertMap.test.js

```diff
@@ -0,0 +1,57 @@
+const timeout = process.env.SLOWMO ? 60000 : 10000;
+const fs = require('fs');
+beforeAll(async () => {
+	path = fs.realpathSync('file://../examples/index.html');
+	await page.goto('file://' + path, {waitUntil: 'domcontentloaded'});
+});
+
+describe('Custom Insert Map', () => {
+	test('Add Custom Insert Map in rich text mode', async () => {
```

Comment:
I'd also love to incorporate any feedback people offer on [this comment](https://github.com/publiclab/PublicLab.Editor/pull/538#pullrequestreview-445890409) in this PR as well. Does that sound all right? Thanks!!